metrics/web/lib/dashboard/presentation/widgets/performance_sparkline_graph.dart

```diff
@@ -29,16 +28,15 @@ class PerformanceSparklineGraph extends StatelessWidget {
     final performanceTheme = MetricsTheme.of(context).metricWidgetTheme;
 
     return Column(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
       children: <Widget>[
         Expanded(
-          child: Padding(
-            padding: const EdgeInsets.all(4.0),
-            child: ExpandableText(
-              DashboardStrings.minutes(performanceSparkline.value),
-              style: TextStyle(
-                color: performanceTheme.primaryColor,
-                fontWeight: FontWeight.bold,
-              ),
+          child: Text(
+            CommonStrings.duration(performanceSparkline.value),
+            style: TextStyle(
+              fontSize: 22.0,
+              color: performanceTheme.primaryColor,
+              fontWeight: FontWeight.bold,
```

Comment:
From my manual/visual testing, it doesn't look that this font-weight is applied, I tried to change it to the normal - same output.

Reply:
We've investigated this issue and it is related to the GoogleFonts package we are currently using, so I created an [issue] (https://github.com/platform-platform/monorepo/issues/385) to replace GoogleFonts with assets. I think it would be better to do it in a separate PR.

Reply:
Thanks @vova-beloded!